internal/handler/message/commands/commands.go

```diff
@@ -1,5 +1,6 @@
 package commands
 
 const (
-	SendRules = "rules"
+	SendRules      = "rules"
+	SendUsefulInfo = "useful_info"
```

Comment:
Є дивні люди (як я наприклад) які пишуть команди повністю, тому краще робити їх коротшими по можливості
```suggestion
	SendUsefulInfo = "useful"
```